histomicstk/cli/ColorDeconvolution/ColorDeconvolution.py

```diff
@@ -76,11 +78,14 @@ def main(args):
                     'yoffset': region.get('top', 0),
                 },
             }],
+            'attributes': {
+                'cli': Path(__file__).stem,
+                'params': vars(args),
+                'version': histomicstk.__version__,
+            },
         }, {
-            'name': 'Deconvolution %s - %s' % (
-                args.stain_3 if args.stain_3 != 'custom' else str(args.stain_3_vector),
-                os.path.splitext(os.path.basename(args.outputAnnotationFile))[0]),
-            'description': 'Used params %r' % vars(args),
+            'name': 'Deconvolution %s ' % (
```

Comment:
What is the logic of changing what these had been named?  

If it makes sense, then we don't want a trailing space as part of the name.

Reply:
I removed that because it was adding the output file name `Color Deconvolution-outputAnnotationFile` to the annotation name which is already present in the newly added `attribute: { params: vars(args)...`. Also, the name is shorter and represents `process: Deconvolution` and `strain: E / H`. I will remove the trailing white spaces in the name.